lib/membrane_file/sink.ex

```diff
@@ -1,48 +1,116 @@
 defmodule Membrane.File.Sink do
   @moduledoc """
   Element that creates a file and stores incoming buffers there (in binary format).
-  """
 
+  When `Membrane.File.SeekEvent` is received, the element starts writing buffers starting
+  from `position`. By default, it overwrites previously stored bytes. You can set `insert?`
```

Comment:
I would add some information to the `SeekEvent` about the role of the `insert?` flag as this event may go through different elements and it will be the first point of contact when it comes to explanations.